Source/WebCore/layout/formattingContexts/inline/text/TextBreakingPositionContext.h

```diff
@@ -56,6 +57,7 @@ inline TextBreakingPositionContext::TextBreakingPositionContext(const RenderStyl
     , wordBreak(style.wordBreak())
     , nbspMode(style.nbspMode())
     , locale(style.computedLocale())
+    , rtl(style.writingMode().isBidiRTL())
```

Comment:
What attribute of being rtl/ltr makes the breaking positions stale? (the explanation should be part of the commit message)